addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -451,7 +451,7 @@ public void sendLogNotification(String message, String icon, String severity) {
                 notificationMessage.put("severity", severity);
                 socket.emit("lognotification", notificationMessage);
             } catch (JSONException e) {
-                logger.error(e.getMessage());
+                logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace